templates/default/sql/dml/0.1.0.sql.erb

```diff
@@ -53,11 +53,6 @@ REPLACE INTO `user_group` (`uid`, `gid`) VALUES (10001, 1007);
 REPLACE INTO variables(`id`, `value`) VALUES ("twofactor_auth", "<%= node['hopsworks']['twofactor_auth'] %>");
 REPLACE INTO variables(`id`, `value`) VALUES ("twofactor-excluded-groups", "<%= node['hopsworks']['twofactor_exclude_groups'] %>");
 
--- kmon
-
-INSERT INTO hosts(hostname, host_ip) VALUES <%= @hosts %> ON DUPLICATE KEY UPDATE cores=NULL;
```

Comment:
We still need the hosts table, don't we?

Reply:
I didn't remove the host table. I just removed the insert - I also changed Hopsworks to create the host if it doesn't exists: https://github.com/logicalclocks/hopsworks-ee/pull/1524

Reply:
that should be fine, we already delete the old hosts and re-register new head when initialising the cluster 
https://github.com/logicalclocks/ec2-init/blob/master/ec2init/plans/preinit/pre_inits.py#L592